Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -213,6 +213,11 @@ public final class XcodeBuildController: XcodeBuildControlling {
             .asSingle()
     }
 
+    public func version() -> Observable<SystemEvent<String>> {
+        let command = ["/usr/bin/xcrun", "xcodebuild", "-version"]
```

Comment:
If I'm not wrong, I think this one returns a new line character at the need that we might want to remove using `.chomp()`.